src/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -24,12 +24,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Debug|AnyCPU'" />
```

Comment:
Instead of updating these guys manually, can we just change Configurations.props and then calling `msbuild /t:UpdateVSConfigurations` from the root so that it updates the solution files as well?